test/sanity/ignore.txt

```diff
@@ -112,7 +112,7 @@ lib/ansible/module_utils/six/__init__.py replace-urlopen
 lib/ansible/module_utils/urls.py pylint:arguments-renamed
 lib/ansible/module_utils/urls.py pylint:disallowed-name
 lib/ansible/module_utils/urls.py replace-urlopen
-lib/ansible/parsing/vault/__init__.py pylint:disallowed-name
+lib/ansible/parsing/vault/editor.py pylint:disallowed-name
```

Comment:
This ignore is for a dummy variable named `_`. I think this would be a reasonable one to rename so we can remove the ignore and prevent accidentally adding more.